packages/lesswrong/server/scripts/mergeAccounts.ts

```diff
@@ -189,6 +189,10 @@ Vulcan.mergeAccounts = async ({sourceUserId, targetUserId, dryRun}: {
 
   // Transfer posts
   await transferCollection({sourceUserId, targetUserId, collectionName: "Posts", dryRun})
+  // Transfer post co-authorship
+  if (!dryRun) {
```

Comment:
Why is this the only check for dryRun?

Reply:
There are some things in this script that already follow this pattern, like `moveUserConversationsToNewUser()`. We could add `dryRun` versions for those things (ideally this script would have those) but I felt like this PR had enough changes.